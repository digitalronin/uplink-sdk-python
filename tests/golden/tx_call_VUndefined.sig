r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 7795212318298116465136769544402885230106677779078863211614147190173901981179
