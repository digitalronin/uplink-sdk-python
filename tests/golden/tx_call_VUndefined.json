{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
                "args": [
                    {
                        "tag": "VUndefined"
                    }
                ], 
                "method": "get"
            }, 
            "tag": "Call"
        }, 
        "tag": "TxContract"
    }, 
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AEw3Nzk1MjEyMzE4Mjk4MTE2NDY1MTM2NzY5NTQ0NDAyODg1MjMwMTA2Njc3Nzc5MDc4ODYzMjExNjE0MTQ3MTkwMTczOTAxOTgxMTc5"
}